app/pages/sections/order-items/catalogue-solutions/order-item/controller.js

```diff
@@ -36,6 +36,7 @@ export const getOrderItemContext = async ({
   formData,
   recipients,
   selectedRecipients,
+  catalogueItemAlreadyExist,
```

Comment:
I think either `catalogueItemExists` or `isExistingCatalogueItem` might be slightly better name.

Reply:
changed to catalogueItemExists